tools/github/hooks/prerun.sh

```diff
@@ -26,13 +28,29 @@ cat <<EOF > warpbuild_body.json
 }
 EOF
 
+echo -e "\nMaking a request to warpbuild..."
+
 # Use wget with retries, retry interval, no certificate check, and exit on failure
 wget --tries=5 --waitretry=2 --retry-connrefused \
   --retry-on-host-error --retry-on-http-error=502 \
+  --content-on-error \
   --no-check-certificate --continue --no-verbose \
   --header="Content-Type: application/json" \
   --header="X-Warpbuild-Scope-Token: $WARPBUILD_SCOPE_TOKEN" \
-  -O - --post-file=warpbuild_body.json \
-  "$WARPBUILD_HOST_URL/api/v1/job" || exit 1
+  -O warpbuild_response.json --post-file=warpbuild_body.json \
+  "$WARPBUILD_HOST_URL/api/v1/job" || exit_code=$? || true
+
+if [ -n "$exit_code" ]; then
+    echo "Failed to send request to warpbuild. Logging response. Exiting..."
+    # check if jq is installed if so then pretty print the json response
+    if ! command -v jq &> /dev/null; then
+        cat warpbuild_response.json
+    else
+        jq . warpbuild_response.json
```

Comment:
if we are printing complete backend error, then we should also structure our error messages/description sent from backend in this api

Reply:
That is structured, right?